pom.xml

```diff
@@ -14,7 +14,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.447</version>
+        <version>1.567</version>
```

Comment:
I realize this is upgrading the version in which unit tests are run against.  Can you confirm my assumption?